third_party/move/move-prover/bytecode-pipeline/src/number_operation.rs

```diff
@@ -217,6 +217,14 @@ impl GlobalNumberOperationState {
             } else {
                 // If not appearing in the pragma, mark it as Arithmetic or Bottom
                 // Similar logic when populating ret_operation_map below
+                let opt_local_type = func_env.get_local_type(i);
```

Comment:
This change is orthogonal to other changes in this PR, right?

Reply:
Oops, I should maybe remove this one.